resources/plugin.yml

```diff
@@ -13,12 +13,12 @@ description: Provides on-the-fly system for permissions system created by Nijiko
 
 permissions:
     groupmanager.*:
-        description: Wildcard permission to grant all permissions.
+        description: A wildcard to grant all permissions.
         default: op
         children:
             groupmanager.op: true
     groupmanager.op:
-        description: User is treated as an op when using the GroupManager commands.
+        description: Given if the user is a server operator

```

Comment:
This is wrong. This permission when granted will allow the user control over GM as if he were OP.
It is granted to ops but thats not the purpose of the node.